backend/src/sharedCopy/applications.ts

```diff
@@ -93,3 +93,21 @@ export function useApplication(type?: number) {
   let app = applications.find(a => a.types.includes(type || 0))
   return app || defaultApp
 }
+
+export const getDateFormatString = () => {
+  const formatObj = new Intl.DateTimeFormat(window.navigator.language).formatToParts(new Date())
+  return formatObj
+    .map(obj => {
+      switch (obj.type) {
+        case 'day':
+          return 'dd'
+        case 'month':
+          return 'MM'
+        case 'year':
+          return 'yyyy'
+        default:
+          return obj.value
+      }
+    })
+    .join('')
+}
```

Comment:
This is great - looks like it's handling international date formatting, but I don't think it belongs in the applications.ts file. How about a new helper file? `/helpers/dateHelper.ts` maybe?